pkg/skuba/actions/cluster/init/constants.go

```diff
@@ -62,5 +62,6 @@ var (
 				Content:  openstackReadme,
 			},
 		},
+		"aws": {},
```

Comment:
Do you think we could at least include a `README.md` file inside the directory?

Reply:
I second the `README.md` this way we can say in the docs "read the readme for you platform" this way documentation can be consistent and handle this with a readme.